### DISCUSSION TITLE
Beginner and willing to contribute to this project

### DISCUSSION THREAD

**Discussion by d14mittal** [eclipse/jkube#1479] [Q&A] [answered]
I was following the video to build the project using the following commands-
mvn k8s:build
mvn oc:build
But I get "No plugin found for prefix 'oc' in the current project and in the plugin groups"
"NO plugin found for prefix 'k8s' in the current project and in the plugin groups"

How to resolve this issue? Am I missing something

**Comment by rohanKanojia** [comment]
In which quickstart did you run this command? Maybe you forgot to specify profile `mvn k8s:build -Pkubernetes` ..

**Comment by sunix** [comment, marked as answer]
The videos are about how to use the maven plugins, not how to build the maven plugin. `mvn clean install` should build the project.
